Essentials/src/com/earth2me/essentials/commands/Commandspeed.java

```diff
@@ -6,13 +6,17 @@
 import org.bukkit.Server;
 import org.bukkit.entity.Player;
 
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 
 import static com.earth2me.essentials.I18n.tl;
 
 
 public class Commandspeed extends EssentialsCommand {
+    private final List<String> types = Arrays.asList("walk", "fly", "1", "1.5", "1.75", "2");
+    private final List<String> speeds = Arrays.asList("1", "1.5", "1.75", "2");
```

Comment:
```suggestion
    private static final List<String> types = Arrays.asList("walk", "fly", "1", "1.5", "1.75", "2");
    private static final List<String> speeds = Arrays.asList("1", "1.5", "1.75", "2");
```

Reply:
I would also suggest using a better range of speeds, such as 1,2,3,4,5 instead of these random decimals

Reply:
or just maybe don't put the numbers lol its kind of weird to suggest specific numbers in general imo